feature/experimental/p4rt/tests/p4rt_election/p4rt_election_test.go

```diff
@@ -670,6 +696,14 @@ func TestArbitrationUpdate(t *testing.T) {
 		// After updating electionID, statusCode also changes
 		// to secondary without a primary
 		test.wantStatus = 5
+		if *deviations.P4rtBackupArbitrationResponseCode {
+			// Change the expected status code to ALREADY_EXISTS for deviant implementations
+			// that send ALREADY_EXISTS instead of NOT_FOUND to secondary clients when there
+			// is no primary
+			test.wantStatus = 6
```

Comment:
nit: can you change the hardcoded numbers to something like: `int32(codes.ALREADY_EXISTS)` and `int32(codes.NOT_FOUND)`